packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -60,6 +60,39 @@ class Scrollable2 extends StatefulWidget {
     if (physics != null)
       description.add('physics: $physics');
   }
+
+  /// The state from the closest instance of this class that encloses the given context.
+  ///
+  /// Typical usage is as follows:
+  ///
+  /// ```dart
+  /// Scrollable2State scrollable = Scrollable2.of(context);
+  /// ```
+  static Scrollable2State of(BuildContext context) {
```

Comment:
maybe make this private? We probably don't want to encourage people to look this class up.